packages/@aws-cdk/aws-apigatewayv2/lib/http/api-definition.ts

```diff
@@ -0,0 +1,189 @@
+import * as s3 from '@aws-cdk/aws-s3';
+import * as s3_assets from '@aws-cdk/aws-s3-assets';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import * as cxapi from '@aws-cdk/cx-api';
+
+import { Node } from 'constructs';
+import { CfnApi } from '../apigatewayv2.generated';
+import { IHttpApi } from './api';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
```

Comment:
This is no longer necessary.

Reply:
I updated the code to address this comment.